cmd/collector/main.go

```diff
@@ -90,20 +90,21 @@ func main() {
 			builderOpts := new(builder.CollectorOptions).InitFromViper(v)
 
 			mBldr := new(pMetrics.Builder).InitFromViper(v)
-			metricsFactory, err := mBldr.CreateMetricsFactory("jaeger-collector")
+			baseFactory, err := mBldr.CreateMetricsFactory("jaeger")
```

Comment:
why are we removing `-collector` suffix? Wouldn't it create potential clashes when used in standalone? And even if not, once these metrics are scraped by a single Prometheus instance, isn't it better to be able to tell apart which binary emitted the metric?